server/athenian/api/controllers/miners/github/contributors.py

```diff
@@ -95,3 +96,70 @@ async def fetch_releases():
         .where(User.login.in_(users)))
     user_details = {r[0]: (r[1], r[2]) for r in user_details}
     return users, user_details
+
+
+@cached(
+    exptime=5 * 60,
+    serialize=marshal.dumps,
+    deserialize=marshal.loads,
+    key=lambda repos, **_: (",".join(repos),),
+)
+async def mine_all_contributors(repos: Collection[str],
+                                db: databases.Database,
+                                cache: Optional[aiomcache.Client]) -> List[dict]:
+    """Discover all developers who made any important action in the given repositories."""
+    async def fetch_prs_authors():
```

Comment:
I was thinking about it, then I decided to make it separate because in addition to not having to filter indeed by timestamp. But it's true that they're actually the same if we put min `datetime` and `now` since I can re-use the `groupby` instead of distinct. I'll work on it 👍

Reply:
done [here](https://github.com/athenianco/athenian-api/pull/344/commits/3fb18a0957309d435ef95024163d4637e3798e48).